coursera-dl/meta.yaml

```diff
@@ -0,0 +1,68 @@
+package:
+  name: coursera-dl
+  version: "3.1.1"
+
+source:
+  fn: coursera-dl-3.1.1.tar.gz
+  url: https://pypi.python.org/packages/source/c/coursera-dl/coursera-dl-3.1.1.tar.gz
+  md5: c2a5089ee5b78874f1f4a9519fb115a8
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # preserve_egg_dir: True
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - coursera-dl = coursera-dl:main
+    #
+    # Would create an entry point called coursera-dl that calls coursera-dl.main()
+
+    - coursera-dl = courseradownloader.courseradownloader:main
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - beautifulsoup4
+    - mechanize
+    - argparse
```

Comment:
`argparse` is part of the standard library in Python 2.7, 3.3 and 3.4. So please remove this dependency.